content/zh-cn/docs/integration-guide.md

```diff
@@ -30,17 +30,17 @@ Let's Encrypt 和 Web PKI 都将随着时间的推移而不断发展。  您应
 
 # 获取更新
 
-要接收有关上述重要更改的小批量更新，请订阅我们的 [API 公告](https://community.letsencrypt.org/t/about-the-api-announcements-category/23836)分类。 这对客户端开发人员和托管提供商都很有用。
+要接收有关上述重要更改的小批量更新，请订阅我们的 \[API 公告\](https://community.letsencrypt.org/t/about-the-api-announcements-category/23836)分类。 这对客户端开发人员和托管提供商都很有用。
```

Comment:
I think the `\` before `[` and `]` are incorrect and should be removed in this file:

![image](https://user-images.githubusercontent.com/1955774/198879684-88a5b9e3-d0c6-47b6-bda4-78c178cce0e7.png)